### PR TITLE
Testing pre-commit

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin Any clue why the pre-commit is having trouble parsing imports after #1? (I.e., by 'pip installable' I also mean importable from the package without errors)

File: setup.py [hunk 1]
```diff
@@ -10,14 +10,14 @@
     long_description = f.read()
 
 setup(
-    name="seidemann-lab-to-nwb",
+    name="seidemann_lab_to_nwb",
```
Comment (h-mayorquin) [inline setup.py]: For this I was following nwb-conversion-tools standard, what's the reasoning behind this?

**Review by h-mayorquin — COMMENTED** [review]
I think that the issue with black is because the class name starts with a number. 

Why did you eliminate the __init__ in `src`?

**Comment by CodyCBakerPhD** [comment]
> I think that the issue with black is because the class name starts with a number.

Ahh, can you try changing that, then? The rest of the stuff was just me trying different things (so feel free to revert)

Yes! I will take care of that.

**Comment by h-mayorquin** [comment]
> > I think that the issue with black is because the class name starts with a number.
> 
> Ahh, can you try changing that, then? The rest of the stuff was just me trying different things (so feel free to revert)
> 

Done, this seems to be working.

**Comment by CodyCBakerPhD** [comment]
Hard to believe GitHub still hasn't enabled cross-user permissions control for editing comments: https://github.com/catalystneuro/seidemann-lab-to-nwb/pull/6#issuecomment-1129256710

Interesting read: https://github.com/isaacs/github/issues/266

File: src/seidemann_lab_to_nwb/embargo22a/embargo22a_metadata.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ NWBFile:
   session_description: >
     A rich text description of the experiment. Can also just be the abstract of the publication.
   institution: My Institution
-  lab: Seidemann
+  lab: seidemann
```
Comment (CodyCBakerPhD) [inline src/seidemann_lab_to_nwb/embargo22a/embargo22a_metadata.yaml]:
```suggestion
  lab: Seidemann
```

**Comment by h-mayorquin** [comment]
> Hard to believe GitHub still hasn't enabled cross-user permissions control for editing comments: [#6 (comment)](https://github.com/catalystneuro/seidemann-lab-to-nwb/pull/6#issuecomment-1129256710)
> 
> Interesting read: [isaacs/github#266](https://github.com/isaacs/github/issues/266)

Is not the first time that I do that x D
Sorry!

**Comment by CodyCBakerPhD** [comment]
> Is not the first time that I do that x D
Sorry!

Nor mine, no prob. Just very annoying GitHub doesn't support any options for controlling it

**Pull request merged**